zerver/migrations/0281_add_message_has_reaction.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.12 on 2020-06-01 06:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0280_userprofile_presence_enabled'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='archivedmessage',
+            name='has_reaction',
+            field=models.BooleanField(db_index=True, default=False),
+        ),
+        migrations.AddField(
+            model_name='message',
+            name='has_reaction',
+            field=models.BooleanField(db_index=True, default=False),
+        ),
+    ]
```

Comment:
This migration needs an `RunPython` migration to set `has_reaction` on every Messages with an associated Reaction.